src/events/cd-booking-create-account.vue

```diff
@@ -113,61 +117,53 @@
         this.formValidated = true;
         return (!this.errors.any() && !!this.recaptchaResponse);
       },
-      submitAccount() {
-        return this.register()
-          .then(this.addChildren)
-          .then(this.joinDojo)
-          .then(this.bookTickets);
+      async submitAccount() {
+        await this.register();
+        await this.addChildren();
+        await this.joinDojo();
+        return this.bookTickets();
       },
-      register() {
+      async register() {
         this.profile = StoreService.load(`booking-${this.eventId}-user`);
         return UserService.register(this.user, this.profile);
       },
-      addChildren() {
+      async addChildren() {
```

Comment:
I believe you've just caught a bug there actually. I should have `await forEachTicket`. The return at the end is pointless too I think.